src/schema/index.js

```diff
@@ -62,6 +62,8 @@ type Transaction {
   createdTime: String!
   blockNum: Int
   blockTime: String
+  gasLimit: Int!
+  gasPrice: String!
```

Comment:
`gasPrice` type should be consistent with `gasLimit`, either `string` or `int`

Reply:
i'll switch `gasLimit` to string. 

i used `gasPrice` as string since there is an issue with graphql receiving `0.0000004` as scientific notation. graphql int does not accept it in that format. i've tried everything to force decimal notation but JS seems to always send it as scientific notation.